src/Angor/Client/Pages/Index.razor

```diff
@@ -20,10 +21,33 @@
                     <Icon IconName="angor-logo" Width="72" Height="72" />
                 </span>
                 <h5 class="card-title text-center">Welcome to Angor</h5>
-                <p class="card-text text-center">Stay in control of your investments with Angor</p>
+                <p class="card-text text-center">Stay in control of your investments with Angor</p>                
             </div>
         </div>
     </div>
 </div>
 
 
+@code{
+    private string btcBalanceInUsd;
+ 
+    protected override async Task OnInitializedAsync()
```

Comment:
so wait we fetch balance only once on startup?

Reply:
removed from index